grype/presenter/cyclonedx/test-fixtures/snapshot/TestCycloneDxPresenter_CycloneDX_Directory_Presenter.golden

```diff
@@ -55,7 +55,7 @@
     </component>
   </components>
   <bd:metadata>
-    <bd:timestamp>2020-08-31T15:39:55-04:00</bd:timestamp>
+    <bd:timestamp>2020-09-17T08:53:32-04:00</bd:timestamp>
     <bd:tool>
       <bd:vendor>anchore</bd:vendor>
       <bd:name>syft</bd:name>
```

Comment:
this should be `grype`, so `NewBomDescriptor()` in  `vuln-extension.go` should be used in `document.go`

Reply:
Although the BOM is produced by Syft, you believe that this should be swapped for Grype?

Reply:
This got updated

Reply:
I thought about the "is it syft or grype" and I think though it's true that syft is producing the work, it would be confusing for someone to run grype and see that it was generated by syft. Open to opinions on this though, I feel there are good arguments either way.

Reply:
I changed it to reflect `grype`. We can always update this later if we feel it isn't working well.